app/generator/to-currency-string.js

```diff
@@ -0,0 +1,7 @@
+const toCurrencyString = (num) => {
+  const numParts = Math.abs(num).toFixed(2).toString().split('.')
+  numParts[0] = numParts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ',')
```

Comment:
Look ahead and look behind -- I can already see the nightmares

Reply:
Totally just copied and pasted from request editor without reading it...

Restored some sanity and added tests.